data/containers/docker-compose.yml

```diff
@@ -2,17 +2,15 @@ version: "3.3"
 services:
 
   nginx:
-    # The REGISTRY is variable and will be replaced
-    image: REGISTRY/library/nginx
```

Comment:
What if use this REGISTRY variable with the new full-path?

Reply:
> What if use this REGISTRY variable with the new full-path?

We want to fully avoid the REGISTRY in this case.

Reply:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/e42002fe2e21841bfa65cf1c133f4f9ad261d49e/tests/containers/compose.pm#L39

Reply:
> https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/e42002fe2e21841bfa65cf1c133f4f9ad261d49e/tests/containers/compose.pm#L39

This command would no thing in this case but I can also remove it.

Reply:
It whould do nothing if you remove REGISTRY :slightly_smiling_face:
But, up to you